css/index.css

```diff
@@ -0,0 +1,236 @@
+body {	
+	width: 100%;
+	font-size: 30px;
+	color: #3cb371;
+	background: url("../img/fondo.jpg") no-repeat center center fixed;
+	background-size:cover;
+}
+header {
+	width: 100%;
+	margin: 0px 0px 20px 0px; 
+	background: rgba(0, 0, 0, 0.8);
+}
+header .fixedmenu {
+	font-family: 'Trade Winds';
+	font-size: 1em;
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-end;
+}
+.fixedmenu a {
```

Comment:
.fixedmenu is repeated, 
Try to add everything in one go.

Reply:
Cambios realizados.